tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -259,3 +274,12 @@ spec:
         - >
           .result == "SUCCESS" or .result == "WARNING" or ($strict | not)
         - "$(results.TEST_OUTPUT.path)"
+
+  volumes:
```

Comment:
Any particular reason a volume, instead of a workspace, was used?

Reply:
No particular reason. Any best practices around this? I'm not sure what's the impact of using one vs. the other.

Reply:
I usually prefer a workspace because it provide more flexibility. While a volume is tied to a particular "storage" type, e.g. ConfigMap, a workspace is not. A workspace can be from a Secret, ConfigMap, PVC, emptyDir, etc. Although most of those would not be applicable in this case, I can see either Secret or ConfigMap being used.

More info here: https://tekton.dev/docs/pipelines/workspaces/

Reply:
Thanks for the explanation! I'll keep that in mind going forward.